### ISSUE TITLE
ParsedSchema should be able to be represented as a dict

### ISSUE THREAD

**Issue by nsheff** [pepkit/pipestat#73]
I need the ParsedSchema object to behave like a dict, so I can return it somewhere where I require a dict.

so I think I need the `__dict__` method to work on it, or have it inherit from dict...

@vreuter do you think you could add some docstrings to the ParsedSchema class so I can get my head around what its role is? And do you think it would be easy to represent it as a `dict` so I can serve it with fastAPI?

Basically I just want an endpoint that returns the schema used to create the pipestat manager.

**Comment by nsheff** [comment]
@vreuter do you have a second you could look at this soon? It's a question that's blocking me and I could dive in, but I think it might take you 5 minutes to solve...

**Comment by vreuter** [comment]
OK, I will check it out
